app/.gitlab-ci.yml

```diff
@@ -512,6 +512,20 @@ release:frontend:
       changes:
       - app/**/*
 
+release:frontend-sourcemaps:
+  needs: ["build:frontend"]
+  stage: release
+  image: $FRONTEND_NEXT_TAG
```

Comment:
I think this should be `$FRONTEND_TAG`; the `next` one is for the dev frontend (which points to `dev-api...`)